Cargo.toml

```diff
@@ -52,10 +53,10 @@ resolver = "2"
 bundler                  = { path = "./nomos-bundler" }
 cfgsync                  = { path = "./testnet/cfgsync" }
 cl                       = { path = "./nomos-core/cl" }
-common-http-client       = { path = "./clients/common-http-client" }
+common-http-client       = { path = "./nodes/nomos-node/client" }
```

Comment:
Why is `common-http-client` inside the `nomos-node` directory now? Isn't it better to put it perhaps still inside the `nodes` folder but in a separate `common` directory?

Reply:
Unless what this really should be is `validator-http-client`?

Reply:
I understood it to be essentially the `validator-http-client`, but we can change this otherwise